### PR TITLE
UI for chat.

### PR DESCRIPTION
**Pull request by nddmanh**
UI for chat:
image

### CODE REVIEW

**Review by lethemanh — APPROVED** [review]

**Review by huyanh7796 — APPROVED** [review]

File: src/modules/playRoom/components/ChatBox.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import imgBtnSend from '../../../assests/images/send.png';
```
Comment (ppppbn) [inline src/modules/playRoom/components/ChatBox.js]: I don't think we should use images as input. It increases the load time for the website and make the UI hard to synchronize with other parts. Instead, try using Antd Icons

File: src/modules/playRoom/pages/PlayRoom.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
-import React, { useState, useEffect } from 'react';
+import { Row, Col} from 'antd';
+import './PlayRoom.scss';
+import ChatBox from '../components/ChatBox';
 
 function PlayRoom(props) {
   return (
-    <div></div>
+    <Row justify="center" gutter={[32, 32]} >
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.js]: If there is only one Col inside the Row. Will you still need gutter?

File: src/modules/playRoom/pages/PlayRoom.scss [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                    width: 40px;
+                    height: 40px;
+                    margin-right: 12px;
+                }
+            }
+        }
+    }
+}
+
+::-webkit-scrollbar{
```
Comment (ppppbn) [inline src/modules/playRoom/pages/PlayRoom.scss]: Please understand that you are trying to modify all the scrollbars that can appear on our website

File: src/modules/playRoom/router/index.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ const authRoutes = [
     name: 'PlayRoom',
     component: PlayRoom,
     meta: {
-      requireAuth: true,
+      requireAuth: false,
```
Comment (ppppbn) [inline src/modules/playRoom/router/index.js]: Here, unchange this

**Pull request merged**